content/docs/getting-started.md

```diff
@@ -18,109 +18,109 @@ redirect_from:
+Kiedy będziesz się już czuć komfortowo z [podstawowymi pojęciami](#main-concepts), a za sobą będziesz mieć trochę zabawy z kodem w Reakcie, możesz być zainteresowany bardziej zaawansowanymi koncepcjami. Ta sekcja zapozna Cię z przydatnymi, acz rzadziej używanymi funkcjonalnościami jak [konteksty (ang. *contexts*)](/docs/context.html) czy [referencjami (ang. *refs*)](/docs/refs-and-the-dom.html).
 
-### API Reference {#api-reference}
+### Dokumentacja API {#api-reference}
 
-This documentation section is useful when you want to learn more details about a particular React API. For example, [`React.Component` API reference](/docs/react-component.html) can provide you with details on how `setState()` works, and what different lifecycle methods are useful for.
+Ta sekcja dokumentacji może okazać się przydatna, jeśli chcesz dowiedzieć się więcej o API Reacta. Na przykład, [dokumentacja `React.Component`](/docs/react-component.html) pozwoli Ci w szczegółach zrozumieć działanie `setState()` oraz jak działają metody z cyklu życia komponentu.
 
-### Glossary and FAQ {#glossary-and-faq}
+### Słownik i FAQ {#glossary-and-faq}
 
-The [glossary](/docs/glossary.html) contains an overview of the most common terms you'll see in the React documentation. There is also a FAQ section dedicated to short questions and answers about common topics, including [making AJAX requests](/docs/faq-ajax.html), [component state](/docs/faq-state.html), and [file structure](/docs/faq-structure.html).
+[Słownik](/docs/glossary.html) zawiera przegląd najczęściej używanych terminów, które napotkasz w dokumentacji Reacta. Istnieje także sekcja FAQ, poświęcona krótkim pytaniom i odpowiedziom na najpopularniejsze tematy, między innymi ["Jak wykonać zapytanie AJAX-owe?"](/docs/faq-ajax.html), ["Co to jest stan komponentu?"](/docs/faq-state.html) oraz ["Jak powinna wyglądać struktura plików?"](/docs/faq-structure.html).
 
-## Staying Informed {#staying-informed}
+## Bądź na bieżąco {#staying-informed}
 
-The [React blog](/blog/) is the official source for the updates from the React team. Anything important, including release notes or deprecation notices, will be posted there first.
+[Blog Reacta](/blog/) jest oficjalnym źródłem aktualizacji i newsów od zespołu Reacta. Wszystkie istotne informacje, w tym notki dotyczące nowych wydań oraz informacje o zdezaktualizowanych elementach, znajdą się tam jako pierwsze.
 
-You can also follow the [@reactjs account](https://twitter.com/reactjs) on Twitter, but you won't miss anything essential if you only read the blog.
+Możesz obserwować [konto @reactjs](https://twitter.com/reactjs) na Twitterze, jednak nie ominie Cię nic ważnego, jeśli będziesz obserwować jedynie bloga.
```

Comment:
```suggestion
Możesz obserwować [konto @reactjs](https://twitter.com/reactjs) na Twitterze, jednak nie ominie Cię nic ważnego, jeśli będziesz śledzić jedynie bloga.
```

Reply:
To już raz odbijałem @jakubdrozdek, z uwagi na to, że na Twitterze "obserwujemy", nie "śledzimy" według tłumaczenia Twittera.

<img width="1220" alt="zrzut ekranu 2019-02-13 o 19 04 40" src="https://user-images.githubusercontent.com/5648158/52733168-55228f80-2fc2-11e9-812d-977117eee726.png">